src/components/views/CertificateForm.tsx

```diff
@@ -46,54 +113,29 @@ const NewCertificate: React.FC = () => {
 
   const handleResetFields = () => {
     setCertificate({
-      supplier: '',
```

Comment:
you are using the same default certificate in few places. Pls defined somewhere constant and use constant